lib/sportradar/api/nfl/play.rb

```diff
@@ -19,6 +20,10 @@ def initialize(data)
         @sequence = data["sequence"]
         @start_situation = Sportradar::Api::Nfl::Situation.new data["start_situation"] if data["start_situation"]
         @statistics = OpenStruct.new data["statistics"] if data["statistics"] # TODO Implement statistics?
```

Comment:
We need to make it easier to work with statistics data; I don't know how consistent it is across the different `statistics_types` (pass, rush, return, etc.) 
I just flat mapped the different types of stats in a given game to see what types were coming back and got:
```
["kick",
 "return",
 "rush",
 "defense",
 "pass",
 "receive",
 "penalty",
 "first_down",
 "down_conversion",
 "field_goal",
 "punt",
 "extra_point",
 "fumble"]
```

So there are at least 13 different types like that, I'll brush up on their documentation to see if we're missing any.